README.md

```diff
@@ -124,6 +124,10 @@ These configurations should be set in an external config file, pointed to with t
 - SPARQL_ENDPOINT (ex: "http://localhost:9082/sparql")
 - XAPIAN_DB_PATH (ex: "/export/data/genenetwork/xapian")
 - TMPDIR
+- SPARQL_USER
+- SPARQL_ENDPOINT (ex: "http://localhost:9082/sparql-auth/")
```

Comment:
`SPARQL_AUTH_ENDPOINT` because we already have another `SPARQL_ENDPOINT`?

Reply:
That `SPARQL_AUTH_ENDPOINT` because that is what Virtuoso provides for INSERTS/UPDATES/DELETES operations in a graph.  To use that end-point, you have to provide a user name and a password.